src/providers/flagsView.ts

```diff
@@ -120,6 +120,8 @@ export class LaunchDarklyTreeViewProvider implements vscode.TreeDataProvider<Fla
 				),
 			],
 			'flagParentItem',
+			'',
+			flag.key,
```

Comment:
The constructors for the types in this file are quite large and we seem to send some empty values to them - is it possible to send one object of options to the constructor so we don't have to specify the optional fields in order?